projects/packages/my-jetpack/_inc/style.module.scss

```
@@ -1,5 +1,15 @@
 :global {
 
+	// Hello Dolly compatibility fix
+	body.jetpack_page_my-jetpack p#dolly {
+		position: absolute;
+		right: 0;
+
+		@media (max-width: 782px) {
+			padding-right: 10px;
```

I would suggest using RTL-aware [logical properties](https://www.w3.org/TR/css-logical-1/)

```suggestion
		inset-inline-end: 0;

		@media (max-width: 782px) {
			padding-inline-end: 10px;
```